src/OrleansRuntime/Placement/DeploymentLoadPublisher.cs

```diff
@@ -178,6 +178,11 @@ private void NotifyAllStatisticsChangeEventsSubscribers(SiloAddress silo, SiloRu
 
         public void SiloStatusChangeNotification(SiloAddress updatedSilo, SiloStatus status)
         {
+            this.ScheduleTask(() => { Utils.SafeExecute(() => this.OnSiloStatusChange(updatedSilo, status), this.logger); }).Ignore();
```

Comment:
Is this breaking isolation? I'm not sure that SystemTargets ever had real isolation within a silo - isn't it more rare that we *don't* call using a mechanism similar to this one here? It seems that the vast majority of the time, calls to a system target follow this pattern:
```C#
scheduler.QueueTask(() => sysTarget.CallMethod(), sysTarget.SchedulingContext).WaitBlah();
```
Where `scheduler` = `OrleansTaskScheduler` and `sysTarget` is a direct .NET reference to the system target instance (not a proxy `GrainReference`).

Regardless, the reason I took this approach is what I mentioned in this comment above: https://github.com/dotnet/orleans/pull/3290#discussion_r133638582